modules/core/24-host/validate.go

```diff
@@ -16,6 +16,10 @@ import (
 // adjusting this restriction.
 const DefaultMaxCharacterLength = 64
 
+// DefaultMaxPortCharacterLength defines the default maximum character length used
+// in validation of port identifiers.
+var DefaultMaxPortCharacterLength = 128
```

Comment:
I chose var instead of const so apps could modify this in the future if they so choose. I don't see a good reason why it needs to be a const?